packages/syntax-core/src/Button/Button.module.css

```diff
@@ -0,0 +1,143 @@
+.button {
+  border: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.buttonGap {
+  gap: 4px;
+}
+
+.button:hover {
+  background-image: linear-gradient(
+    to bottom,
+    rgba(0, 0, 0, 0.1) 0%,
+    rgba(0, 0, 0, 0.1) 100%
+  );
+  transition-duration: 0.2s;
+  cursor: pointer;
+}
+
+.button:focus-visible {
+  background-image: linear-gradient(
+    to bottom,
+    rgba(0, 0, 0, 0.2) 0%,
+    rgba(0, 0, 0, 0.2) 100%
+  );
+  box-shadow: 0 0 0 4px #000;
+  outline: solid 2px #fff;
+}
+
+.button:active {
+  background-image: linear-gradient(
+    to bottom,
+    rgba(0, 0, 0, 0.2) 0%,
+    rgba(0, 0, 0, 0.2) 100%
+  );
+  transform: scale(0.97);
+  transition-duration: 0.2s;
+}
+
+.button:disabled {
+  filter: opacity(30%);
+  background-image: none;
+  transform: none;
+  cursor: auto;
+}
+
+.fullWidth {
+  width: 100%;
+}
+
+.sm {
+  min-width: 52px;
+  height: 32px;
+  padding-left: 12px;
```

Comment:
would these padding-left/rights be better off as `padding: 0 12px`? i don't see that we're explicitly setting padding top/bottom anywhere

Reply:
@somethiiing Yes, updated